### DISCUSSION TITLE
Get list of followers

### DISCUSSION THREAD

**Discussion by nandaccio** [Xwilarg/TwitterSharp#10] [Q&A] [answered]
I quickly went through the source code, but I did not find any way to get the list of followers of a Twitter account. Is it doable with TwitterSharp?

Thanks!

**Comment by Xwilarg** [comment, marked as answer]
Hey,

It's not possible yet but I'll have a look at it around next week

**Reply by r3l1c7** [reply]
any update on this?

https://developer.twitter.com/en/docs/twitter-api/v1/accounts-and-users/follow-search-get-users/api-reference/get-followers-list

**Reply by Xwilarg** [reply]
Hey, sorry for the late answer, I've been quite busy so I just began working on it

About your link, this project use the V2 of the API so it would be this one: https://developer.twitter.com/en/docs/twitter-api/users/follows/api-reference/get-users-id-following

**Reply by Xwilarg** [reply]
Well it's finally here in the version 1.1.0 of the library: https://www.nuget.org/packages/TwitterSharp/1.1.0 \
You can find an example in the README: https://github.com/Xwilarg/TwitterSharp#get-the-users-that-someone-follow

**Reply by r3l1c7** [reply]
> Well it's finally here in the version 1.1.0 of the library: https://www.nuget.org/packages/TwitterSharp/1.1.0 You can find an example in the README: https://github.com/Xwilarg/TwitterSharp#get-the-users-that-someone-follow

Thank you for your time and effort.
